app_controller.php

```diff
@@ -64,8 +64,10 @@ function app_controller()
                 }
             }
             
-            $result = "<link href='".$path."Modules/app/Views/css/app.css?v=".$v."' rel='stylesheet'>";
-            $result = '<script src="'. $path . 'Modules/app/Views/js/app.js?v="'.$v.'"></script>';
+            $result = "\n<!-- global app css and js -->";
+            $result .= "\n" . '<link href="' . $path . 'Modules/app/Views/css/app.css?v=' . $v . '" rel="stylesheet">';
+            $result .= "\n" . '<script src="' . $path . 'Modules/app/Views/js/app.js?v=' . $v . '"></script>';
```

Comment:
should `type="text/javascript" `be added between `<script src=`

Reply:
no, this is not required anymore 
> https://html.spec.whatwg.org/multipage/scripting.html#attr-script-type